src/mmrelay/matrix/command_bridge.py

```
@@ -0,0 +1,246 @@
+
+    if not require_mention and any(
+        re.match(bare_pattern, body, re.IGNORECASE) for body in bodies if body
+    ):
+        return True
+
+    mention_parts: list[str] = []
+    for ident in (facade.bot_user_id, facade.bot_user_name):
+        if ident:
+            try:
+                mention_parts.append(re.escape(str(ident)))
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)